posts/do-all-roads-lead-to-rome.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Do all roads lead to Rome?
+created: 08/24/2020
+description: Learn what Rome is, how it fits into the JavaScript ecosystem and my thoughts about it... Will Rome replace all the current tooling?
+tags: webdev, javascript
+cover_image: /images/cover-images/17_cover_image.jpg
+cover_image_mobile: /images/cover-images/17_cover_image_mobile.jpg
+cover_image_vert: /images/cover-images/17_cover_image_vert.jpg
+cover_color: '#BFB094'
+---
+
+We'll see what [Rome](https://github.com/romefrontend/rome) is, how it fits into the JavaScript ecosystem and my thoughts about it. **Will Rome replace all the current tooling?** Let's see.
+
+## What is Rome?
+
+[**Rome**](https://github.com/romefrontend/rome) is a CLI that unifies the JavaScript and Typescript development toolchain: bundling, compiling, documentation generation, formatting, linting, minification, testing, type checking, etc. It tries to be a "**one-in-all**" solution, being able to do all these tasks under one CLI.
```

Comment:
Rome is a toolchain, not only a CLI. It can be used via CLI but also via VSCode extension. In the future also via browser

Reply:
I didn't know it about the VSCode extension! Thanks a lot!